articles/tech-resources/change-log-sparkpost-enterprise.md

```diff
@@ -7,6 +7,15 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
 
+### (Week of) 28 July, 2017 
```

Comment:
"Accepted"? I dunno, I created this page and have maintained it from the start so I accept it. :-). Upgrades to Enterprise Tenants will happen that week so they happen on different days for different customers. I can change it to be that Friday if "week of" seems odd.